iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateCouplingDevice.java

```diff
@@ -94,7 +94,7 @@ public void apply(Network network, NamingStrategy namingStrategy, boolean throwE
             LOGGER.error("Voltage level associated to {} or {} not found.", busOrBbs1, busOrBbs2);
             notFoundBusOrBusbarSectionVoltageLevelReport(reportNode, busOrBbsId1, busOrBbsId2);
             if (throwException) {
-                throw new PowsyblException(String.format("Voltage level associated to %s or %s not found.", busOrBbs1, busOrBbs2));
+                throw new PowsyblException(String.format("Voltage level associated to %s or %s not found.", busOrBbs1.getId(), busOrBbs2.getId()));
```

Comment:
Can you add a unit test on this exception?

Reply:
it is complicated to test 
```java
 VoltageLevel voltageLevel1 = getVoltageLevel(busOrBbs1, reportNode, throwException);
        VoltageLevel voltageLevel2 = getVoltageLevel(busOrBbs2, reportNode, throwException);
        if (voltageLevel1 == null || voltageLevel2 == null) {
```
it means we have a busbar that has no voltageLevel, I do not know how to create that

Reply:
OK, then we can keep it like this. This is not crucial.